### DISCUSSION TITLE
BuildiBase Button on each row to table to populate form

### DISCUSSION THREAD

**Discussion by rorystpats** [Budibase/budibase#10079] [Help]
Our company has a google sheets document containing orders which come in during the day. This form can send collection messages to our customers when a particular field is populated or 'not empty'. We wish to use BuildiBase to create a User Interface for our staff to view in real time the number of orders coming in and easily have the option to select READY or HIDE which would populate the google form accordingly.

I have assembled a preview app and have achieved a two way data stream between BuildiBase and Google Sheets. However in order to activate the message send staff must click into each row and manually update the 'READY' column with some data to enable the automatic message.

I wanted to install a large button which is linked to each row in the form and would hence save staff time from having to click into each row individually as there is many orders per day.

I have tried a checkbox from Google sheets end and this is not recognized by BuildiBase. I also noted there is a checkbox option in BuildiBase which is connected to a form but I am unsure how to connect this to my existing data and ensure the message is sent to the correct person as this form is constantly being updated every day.

Any advice on this matter would be surely appreciated.

**Comment by melohagan** [comment]
Hey @rorystpats 

You can add buttons to each table row if that is what you need: https://docs.budibase.com/docs/button#adding-buttons-to-table-rows

As noted, you cannot add buttons to Table block rows, however you can easily [Eject](https://docs.budibase.com/docs/blocks#ejecting-blocks) the block to get to the underlying table.
